worknenjoy/gitpay
# Creation of retry button of order payment and bug fix

## Description

> Implemented a button when the order payment was failed. Fixed a bug in paypal payment checkbox.

## Changes

- [x] Task List

## Status

> On review

## Related Issue

> #256 

## Impacted Area

> Task List
> Order Payment (Paypal)

## Steps to test

- `git clone git@github.com:worknenjoy/gitpay.git`
- `cd gitpay/frontend`
- `npm install`
- `npm run dev`
- Go to http://localhost:8082/
- Create a task
- Add a bounty with Paypal
- Cancel the checkout
- You will see a pending payment with a retry button
- Click on retry button
